nixos/modules/services/x11/display-managers/default.nix

```diff
@@ -171,6 +171,15 @@ in
 
     services.xserver.displayManager = {
 
+      enable = mkOption {
```

Comment:
I think `select` makes more sense than any other name.  `enable` is too much tied to a boolean value, while `select` clearly suppose a finite set of options.  I would avoid the passive form here, as this is an input, and not a result.